lapce-ui/src/title.rs

```diff
@@ -97,7 +105,19 @@ impl Widget<LapceWindowData> for Title {
                     )
                 }
             }
-            _ => {}
+            _ => {
+                #[cfg(target_os = "windows")]
+                {
+                    ctx.submit_command(
```

Comment:
Is this for when the ```custom_titlebar``` is toggled in settings? It's better to put in Update and run this when the settings is actually changed.

Reply:
moved to `update`